### ISSUE TITLE
Windows 11 Ver 21H2 build 22000.527 ntvdm installer, InstallShield, Adobe Installer, libreoffice and skype crash links error (?)

### ISSUE THREAD

**Issue by MufasaFasagan** [leecher1337/ntvdmx64#173]
Hi, after the last update of Windows 11 Ver 21H2 build 22000.527, a LOT of problems came out of nowhere, maybe it's a cpu related with my amd ryzen 5600x...

1: ntvdm the "inst-sym.bat" dosen't work, i've to copy manually the "symbols" folder in to "C:\Windows\System32\symbols" to make it work. the message says "I did not find the comm.sym in the symbols folder"
![2022-02-27_140218](https://user-images.githubusercontent.com/88245975/155883932-c110c036-c983-4a46-bbbd-2b03c147c7cc.png)
2: i cannot install/uninstall programs with "new" installshield or msi packages
3: libre office simply crash
4: adobe installer (acrobat, photoshop and audition) gives me a generic error, it simply refuses to install
5: skype crashes when i'm trying to open a link, chat or links like "about skype"
6: EVERY dos program has the same error
![2022-02-27_142602](https://user-images.githubusercontent.com/88245975/155884292-4ecefbca-7879-47b8-98ed-5ffa7839b998.png)

Without ntvdm, I don't have NONE of the problems mentioned above

**Comment by leecher1337** [comment]
Hi, no issues here so far, updated to mentioned verison without issues:

![image](https://user-images.githubusercontent.com/6495842/155893388-d302e361-620e-4e43-8ada-092ca67c85d1.png)

Are you sure that you are using the most recent loaders (see https://github.com/leecher1337/ntvdmx64/issues/167 )?
Amy CPU related errors were fixed in latest update that could cause problems like those you mentioned

Regarding installation of symbols for all DOS/WIN16 applications, installation is not necessarily needed for operation, it's just if you want to debug NTVDM. Not sure, why comm.sym didn't get built when you compiled, maybe check logfiles that get produced before the final keypress when autobuild finishes. Maybe you did a FREE-build, not a checked build where there are no usable debug symbols available (for debugging the ntvdm and components, always use the checked build)

**Comment by MufasaFasagan** [comment]
I've used a old already-compiled copy (free thing I guess), however I don't remember how to compile the new version... I've made a video when skype closes, this happen with vmware, vegas too...same behaviour https://i.imgur.com/1JqmU0Z.mp4
For the "symbol" I thought it was mandatory to install it :/

**Comment by leecher1337** [comment]
No support will be provided for old versions, just run autobuild with build dependencies to get a new version. 
https://github.com/leecher1337/ntvdmx64/blob/master/ntvdmpatch/doc/autobuild.txt

If you just want to update the loaders (these are mandatory to update), you can replace them with the precompiled dlls from this repository, if - for whatever reason - you want to keep your old installer, but as mentioned, there will be no support for obsolete versions.

**Comment by MufasaFasagan** [comment]
> No support will be provided for old versions, just run autobuild with build dependencies to get a new version. https://github.com/leecher1337/ntvdmx64/blob/master/ntvdmpatch/doc/autobuild.txt
> 
> If you just want to update the loaders (these are mandatory to update), you can replace them with the precompiled dlls from this repository, if - for whatever reason - you want to keep your old installer, but as mentioned, there will be no support for obsolete versions.

Ok, i've built a new version, now it's even worse, skype it dosen't work anymore, and i've two new errors
![2022-02-27_203625](https://user-images.githubusercontent.com/88245975/155897107-df7905f8-84ad-4ecc-a78c-e4e62901580f.png)

"Access Denied"

![2022-02-27_203716](https://user-images.githubusercontent.com/88245975/155897111-cf010b28-1af5-4e62-b3f3-ac5bb5c50e42.png)
now the accented words have gibberish character, they should be "più" and "funzionalità"

**Comment by leecher1337** [comment]
Check debug output of loader with Debugview, maybe you are missing some necessary loader symbols.

**Comment by MufasaFasagan** [comment]
> Check debug output of loader with Debugview, maybe you are missing some necessary loader symbols.

So, i've rebuilt (6 times now with the new ntvdmpatch) with your youtube video, NOW 16bit apps works again, but the skype and dos crashes persist

skype error
![skype error](https://user-images.githubusercontent.com/88245975/155900319-c6000770-baf9-4076-92d6-a5acb67cb1e4.png)

dos error
![dos error](https://user-images.githubusercontent.com/88245975/155900339-4477db70-2aa4-47c3-83f9-f1b2347e8dcd.png)

**Comment by leecher1337** [comment]
There you have your problem:

"Cannot resolve LdrpInitializeProcess"
--> NTDLL.DLL Symbols wrong/unresolvable
DBGHELP: "C:\Windows\Symbols\wntdll.pdb\F1D5....." - drive not ready

May be due to a 0byte NTDLL symbol file (wntdll.pdb), this sometimes 
happens if symbol downloading fails. Just delete the given
directory from the error message (The one starting with F1D5...)


If - for whatever reason - this also doesn't help, delete all symbols from 
\windows\Symbols directory (only the directories with the symbol files in 
there, not the other files) and remove the following registry keys to enforce 
redownloading all symbols:

HKEY_LOCAL_MACHINE\SOFTWARE\ldntvdm
HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\ldntvdm

**Comment by MufasaFasagan** [comment]
Hi, i've tried the last solution, but it didn't work.

dos error with last build (the dos error has now the same error with the old build from january, maybe there is a connection...)
![DOS](https://user-images.githubusercontent.com/88245975/155987126-61fee158-9971-4939-9da2-4d9ee056deb4.png)
skype error with last build
![SKYPE ERROR 2](https://user-images.githubusercontent.com/88245975/155987133-e363fa20-321b-4dc4-90d4-0fc5ceba1e54.png)
skype with the old version from january works, but crashes when I open a link
![SKYPE ERROR OLD VERSION JANUARY](https://user-images.githubusercontent.com/88245975/155987170-fc50afbf-6705-44d5-856d-3b6c8c624e43.png)

I give up, maybe the culprit is this s*****y amd 5600x or win11, something broke, and pretty hard, a fresh install of win11 sadly has the same results. My favorite feature of ntvdm was the ability to see icons with 16bit applications, :/
next time i'll buy an intel cpu...

**Comment by leecher1337** [comment]
The faulty wntdll.pdb is still there, as you can see.. So you deleted this file and it was recreated and still has 0 bytes and is still incomplete? 
Did you check the file? Did you ensure that you deleted it?

**Comment by MufasaFasagan** [comment]
So, i've deleted the file "wntdll.pdb", reinstalled ntvdm and...that file is ALWAYS 0 byte, so i deleted the "wntdll.pdb" again, then i've tried the programs...NOW SKYPE and DOS WORKS WTF?!? I've done it at least 10 times! Now the file has some "weight"
![2022-02-28_201843](https://user-images.githubusercontent.com/88245975/156044933-a30f3a6e-6d4a-4920-9d69-93b3c5d1971d.png)
![2022-02-28_200959](https://user-images.githubusercontent.com/88245975/156043820-9cf5f1ef-b31f-4357-b825-bcc45ece32c3.png)

Oh...i notice this when i install ntvdm with admin privilege, another way to download complete "symbols"? Last String has this comment "ERROR: i can't find a specific file" (????????????????????)
![2022-02-28_192025](https://user-images.githubusercontent.com/88245975/156038817-3eb40986-cf6d-44b3-b4d4-cfb8fe441138.png)

So, in short...there is something preventing me from downloading the symbols wtf? Thanks leecher1337, for now we have made progress!
